enhancements/substitutes-for.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Substitutes-For-Graph-Replacements
+authors:
+  - "@bparees"
+reviewers:
+  - "@ecordell"
+  - "@kevinrizza"
+  - "@lgalleti"
+approvers:
+  - TBD
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: provisional
+---
+
+# Bundle Substitution
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
```

Comment:
i'm not clear on how that question relates here.

for now the version field is whatever the operator teams decide to make it be/mean.  If they opt into freshmaker, then they need to accept that it will periodically get bumped outside of their control.  But that's 
1) their choice to opt into
2) specific to freshmaker and not a part of the substitutes-for behavior itself.